CURSOR.md

```
@@ -0,0 +1,235 @@
+# When running inside Docker network, baseURL is the prod service name
+NEXT_PUBLIC_PROD_CONTAINER_API_URL=http://prod:3001
+
+# Optional: CloudFront continuous deployment headers (used by tests/audits)
+NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME=X-My-Header
+NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_VALUE=some-value
+```
+Notes:
+- When running commands on the host (without Docker), prefix with `CI=1`.
+- `start-prod` creates/uses an external Docker network and runs a static server for `out` on port `NEXT_PUBLIC_PROD_PORT`.
+
+### Scripts and automation
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)